docs/source/markdown/podman-artifact-add.1.md.in

```diff
@@ -21,6 +21,10 @@ added.
 
 @@option annotation.manifest
 
+#### **--append**, **-a**
```

Comment:
if we decide that --type and --append cannot be used together, please update the docs